project03.py

```diff
@@ -232,7 +232,14 @@ def verifyDeathBefore150YearsOld(person):
         age = death_unix - bday_unix
     years_in_seconds = 150 * 365 * 24 * 60 * 60
     return age <= years_in_seconds
-
+  
+def marriageAfter14(person1, person2):
+    #check if both people are above 14 years of age to marry
+    if (person1['age'] > 14 and person2['age'] > 14):
```

Comment:
This doesn't solve the user story, it must verify that **marriage** happens between of-age individuals. You are also not actually verifying the input gedcom file in main